app/assets/stylesheets/components/_top-bar.scss

```diff
@@ -52,6 +34,18 @@
         //@include hamburger($color, $color-hover, $width, $height, $weight, $bars);
         @include hamburger($primary-color, $black, rem-calc(26), rem-calc(21), rem-calc(3));
       }
+
+  &__nav-wrapper {
+    width: 100%;
+
```

Comment:
I generally leave a line between 'design properties' and 'positioning properties' so if I change it here I will need to do it for every class in all files.